content/webapp/pages/event.js

```diff
@@ -43,6 +43,15 @@ type State = {|
   scheduledIn: ?UiEvent,
 |};
 
+function haveIcon(iconName) {
```

Comment:
Why bother with this and not just put it in the component?
If we want to keep it though can we type it?